aquarium/src/cg/group4/aquarium/Configuration.java

```diff
@@ -0,0 +1,95 @@
+package cg.group4.aquarium;
+
+import com.badlogic.gdx.Gdx;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Data class containing the configuration required for the Aquarium.
+ */
+public class Configuration {
+
+    /**
+     * Tag for debugging & logging.
+     */
+    final protected String cTag = this.getClass().getSimpleName();
+
+    /**
+     * Loaction of the server configuration.
+     */
+    final protected String cConfigFile = "aquarium/server.cfg";
+
+    /**
+     * Reader of the configuration file.
+     */
+    protected ConfigurationReader cAquariumSettingsReader;
+
+    /**
+     * Server Host IP cAddress.
+     */
+    protected String cHost;
+
+    /**
+     * Server Host cPort.
+     */
+    protected Integer cPort;
+
+    /**
+     * Initializes the configuration data class.
+     */
+    public Configuration() {
+        try {
+            readConfig(cConfigFile);
+        } catch (IOException e) {
+            Gdx.app.log(cTag, e.getMessage());
+        }
+    }
+
+    /**
+     * Reads the host/ip:cPort configuration using a ConfigurationReader.
+     * @param file Location of the config file.
+     * @throws IOException thrown on failure of file operations
+     */
+    public void readConfig(final String file) throws IOException {
+        cAquariumSettingsReader = new ConfigurationReader(new Scanner(new File(file)));
+        ArrayList a = cAquariumSettingsReader.readSettings();
```

Comment:
You could have the Reader do this functionality and just ask the reader (for example) .getHost() and .getPort() instead of having to cast in the configuration.